doc/user/zebra.rst

```diff
@@ -351,6 +360,143 @@ default) should the specified gateways not be reachable. Eg:
 
 .. _multicast-rib-commands:
 
+VRF Commands
+============
+
+Currently, the user has the possibility to configure VRFs. VRF is a way to
+separate networking contexts on the same machine. Those networking contexts
+stick to separate interfaces, thus making it possible to associate one interface
+to a specific VRF.
+VRF can be used, for example, when instantiating per enterprise networking
+services, without having to instantiate the physical host machine or the
+routing management daemons for each enterprise. As a result, interfaces are
+separate for each set of VRF, and routing daemons can have their own context
+for each VRF.
+
+This conceptual view introduces the `Default VRF` case. If the user does not
+configure any specific VRF, then by default, the user will however configure
+the `Default VRF`.
+On the Zebra context, this can be done when being in configuration mode, when
+configuring a static route `ip route NETWORK GATEWAY`.
```

Comment:
Please cross-reference the relevant CLI command like so:
```
:clicmd:`ip route NETWORK GATEWAY`
```
This will generate a link to the command definition.
http://frrouting.readthedocs.io/projects/dev-guide/en/latest/workflow.html#cli-commands

Reply:
ok